SecurityAgent/SecurityAgent.h

```diff
@@ -168,8 +168,9 @@ namespace Plugin {
         std::unique_ptr<TokenDispatcher> _dispatcher; 
         Core::ProxyType<RPC::InvokeServer> _engine;
         string _testtoken;
+        string _servicePrefix;
 
-        static constexpr TCHAR TestTokenContent[] = _T(R"--({ "url": "https://test.url.com", "user":"Test" })--");
```

Comment:
Should this not alos be enclosed in the SECURITY_TESTING_MODE preprocessor statements ?

Reply:
done